### ISSUE TITLE
Detected as a virus. Ms Edge

### ISSUE THREAD

**Issue by xristosv1234** [WujekFoliarz/DualSenseY-v2#1]
(Trojan:Script/Wacatac.B!ml)

I dont think it is but yeah maybe look into it

**Comment by WujekFoliarz** [comment]
Win defender loves to false flag my programs and i have no idea how to combat this. I have to leave it as is for now

**Comment by SiTWulf** [comment]
Not detected in latest 20

**Comment by WujekFoliarz** [comment]
Maybe Microsoft removed the false positive cuz I reported it like 2 weeks ago

**Comment by lightstylez** [comment]
Windows Defender instantly blocked it.
![Screenshot 2024-12-03 125215](https://github.com/user-attachments/assets/87f04fe9-f1e6-47b4-9245-bd9760b0028d)

**Comment by WujekFoliarz** [comment]
![{8E94D459-7278-4267-A6D9-15FF38A295DF}](https://github.com/user-attachments/assets/5dfae279-2601-4c71-a022-3e9901ef1d10)

maybe that helped

**Comment by lightstylez** [comment]
Yeah, windows no longer flags it. Still detected on some AVs but most say its clean. 
![Screenshot 2024-12-04 150416](https://github.com/user-attachments/assets/5d50b37a-0411-409e-addf-404ba486b166)

**Comment by WujekFoliarz** [comment]
I don't think anyone uses these no-name AVs these days so whatever. Win Defender's what's important

**Comment by SiTWulf** [comment]
Once again download (release 21) is detected as virus. (MS EDGE)

**Comment by WujekFoliarz** [comment]
![{906280E3-9BEE-4395-B5B5-2411C0A991F5}](https://github.com/user-attachments/assets/7721b48b-f3a4-442a-8f45-58c1166c5116)

I guess I will have to do this every time I make a new update

**Comment by WujekFoliarz** [comment]
![{19F956C9-C579-44D8-B260-3CDAAAD22BB1}](https://github.com/user-attachments/assets/2b32e66d-ac69-4a31-a107-242f83e28271)

The detections mainly come from these two

**Comment by WujekFoliarz** [comment]
Maybe I should rename the actual app .exe file to "DSX.exe". Seems like the most logical fix

**Comment by SiTWulf** [comment]
Once again download (release 32) is detected as virus. (MS EDGE

**Comment by WujekFoliarz** [comment]
![{62F909A0-B079-4F43-9EE8-06444EBB33D4}](https://github.com/user-attachments/assets/f00ea9af-3535-44aa-87da-71c9556fd00c)

**Comment by MohandL3G** [comment]
and for me to (release 37) is detected as virus by Windows Defender

**Comment by WujekFoliarz** [comment]
![Image](https://github.com/user-attachments/assets/55d5cc33-b177-4b60-b424-46af198ceffc)

**Comment by WujekFoliarz** [comment]
![Image](https://github.com/user-attachments/assets/b74a8d87-befd-4a54-9354-c39a199b0cd4)

**Comment by mchar7** [comment]
Latest release (50) flagged by Windows Defender and 27/69 on [VirusTotal](https://www.virustotal.com/gui/file/54b4248d40a3bccb50dda698586db54f357c7fc0b1f8cab6a121b1bf297e6aec/behavior) including decent AVs like Kaspersky, MalwareBytes, and Bitdefender:

<img width="2626" height="1855" alt="Image" src="https://github.com/user-attachments/assets/6ba2baad-5dab-4d13-90a7-61b67025fd87" />

Also makes UDP connection to `maluch.mikr.us:30151`? I can't see any reference to that in the source.

Most detections seem to be coming from the embedded HidHide.exe (23/72), Updater.exe (9/72), and BTControl.exe (6/72) and not DualSenseY.exe (1/72).

**Comment by WujekFoliarz** [comment]
<img width="387" height="81" alt="Image" src="https://github.com/user-attachments/assets/bada522a-c853-46f6-9ef2-36bb0ec7f468" />

Submitted, I'll just do this every time I upload new version, I only care about windows defender

**Comment by WujekFoliarz** [comment]
<img width="1426" height="292" alt="Image" src="https://github.com/user-attachments/assets/c0f61506-2f7f-43de-a3fe-60758257ba32" />

Oh my god bruh
